### PR TITLE
simple SMTP output bot

### PR DESCRIPTION
**Pull request by wagner-certat**
Very simple, highly configurable :)

### CODE REVIEW

File: docs/Bots.md [hunk 1]
```diff
@@ -801,6 +801,40 @@ from your installation.
 
 * * *
 
+# SMTP Output Bot
+
+Sends a MIME Multipart message containing the text and
```
Comment (aaronkaplan) [inline docs/Bots.md]: ... and the event?

File: docs/Bots.md [hunk 2]
```diff
@@ -801,6 +801,40 @@ from your installation.
...
+#### Information:
+* `name:` smtp
+* `lookup:` no
+* `public:` yes
+* `cache (redis db):` none
+* `description:` Sends events via SMTP
+
+#### Configuration Parameters:
+
+* `fieldnames`: comma separated string or list of strings
```
Comment (aaronkaplan) [inline docs/Bots.md]: maybe be clear that this is a whitelist

File: docs/Bots.md [hunk 3]
```diff
@@ -801,6 +801,40 @@ from your installation.
...
+* `mail_from`: string. Supports formatting, see below
+* `mail_to`: string of email addresses, comma separated. Supports formatting, see below
+* `smtp_host`: string
+* `smtp_password`: string or null, Password for authentication on your SMTP server
+* `smtp_port`: port
+* `smtp_username`: string or null, Username for authentication on your SMTP server
+* `ssl`: boolean
```
Comment (aaronkaplan) [inline docs/Bots.md]: http_verify ?

File: docs/Bots.md (hunk 1 above)
Comment (aaronkaplan) [inline docs/Bots.md]: Add a text that this will send an email for *every individual* event that it receives

**Comment by wagner-certat** [comment]
Pushed documentation fixes and certification validation

**Pull request merged**